source/fundamentals/connection/connect.txt

```diff
@@ -18,19 +26,19 @@ example shows each part of the connection URI:
 .. figure:: /includes/figures/connection-string-parts.png
    :alt: Each part of the connection string
 
-In this example, for the protocol, we use ``mongodb+srv`` which specifies the
-:manual:`DNS Seedlist Connection Format
-</reference/connection-string/#dns-seedlist-connection-format>`. This indicates
-that the hostname following it corresponds to the DNS SRV record of your
-MongoDB instance or deployment. If your instance or deployment does not have a
-DNS SRV record, use ``mongodb`` to specify the :manual:`Standard Connection
-Format </reference/connection-string/#standard-connection-string-format>`.
+In this example, we use ``mongodb`` for the protocol, which specifies
+the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.
+
+.. tip::
+
+   You can also use the :manual:`DNS Seed List Connection Format
+   </reference/connection-string/#dns-seed-list-connection-format>` if
+   you want more flexibility of deployment and the ability to change the
+   servers in rotation without reconfiguring clients.
```

Comment:
I: "also" sort of suggests it's an add-on to the standard format, rather than an alternative

```suggestion
   If your instance or deployment has a DNS SRV record, you can use the 
   :manual:`DNS Seed List Connection Format
   </reference/connection-string/#dns-seed-list-connection-format>` instead.
   This format offers more flexibility of deployment and lets you change the
   servers in rotation without reconfiguring clients.
```